OpenSim/Simulation/SimbodyEngine/Coordinate.h

```diff
@@ -221,6 +221,11 @@ OpenSim_DECLARE_CONCRETE_OBJECT(Coordinate, ModelComponent);
     SimTK::MobilizedBodyIndex getBodyIndex() const { return _bodyIndex; };
     /**@}**/
 
+    /* For internal consistency checking. Returns the user-specified MotionType
+       serialized with older model files (prior to 4.0) if one is provided, other-
```

Comment:
maybe "older model files (prior to 4.0)" -> "pre-4.0 model files", so that this comment doesn't get stale?